qemu/tests/cfg/multi_disk_random_hotplug.cfg

```diff
@@ -31,6 +31,9 @@
                 stg_params = "fmt:virtio,virtio_scsi,usb2"
             usbs += " ehci"
             usb_type_ehci = usb-ehci
+            Host_RHEL.m6:
+                usbs= "ehci"
```

Comment:
original configuration has more usb controllers,  after your update only one control 'ehci' for rhel6 host, why?